docs/docs/user/core-concepts/components/kubernetes-app.md

```diff
@@ -22,11 +22,11 @@ Can be used to deploy any app in Kubernetes using Helm, for example, a REST serv
 
 #### deploy
 
-Deploy using Helm.
+Do nothing.
 
 #### destroy
 
-Uninstall Helm release.
+Do nothing.
```

Comment:
question for team: 
should we leave the implementation of these empty or provide a default implementation that does a simple _apply_ / _delete_ using the Kubernetes API or kubectl?

Reply:
Since the empty ones are already in `PipelineComponent` I'd say provide a default implementation.

Reply:
Unless we need to deploy anything with Kubernetes API, I would keep it like that. If the feature is needed I would add it in a separate PR. 
I propose using the Kubernetes API and avoid using the `kubectl` command (wrapper).

Reply:
agreed, could be separate PR once there's a need